### ISSUE TITLE
Need help capturing replaceResultResponse from Blackboard

### ISSUE THREAD

**Issue by gtheilman** [IMSGlobal/LTI-Tool-Provider-Library-PHP#42]
I'm having inconsistent success getting scores passed back to the Tool Consumer' gradebook.
*   Works fine with emulators at   http://ltiapps.net/test/tc.php .   Scores get added to gradebook with no problems
*  Scores are posted when using a newly created instance of Canvas
*  Can't get scores to pass back to my campus Blackboard server
*  Can't get scores to pass back to Canvas instances hosted by Instructure.

I've confirmed that the Blackboard server can accept scores passed back from other tool providers.

What I think I need is the ability to see what replaceResultResponse is being returned by Blackboard.  If it is returning an error message in the XML, I can probably determine the problem.

It looks like if I might be able to modify the doLTI11Service function in ResourceLink.php to capture the response from Blackboard.   But I'm unable to figure out a strategy for capturing the replaceResultResponse.

Any suggestions would be appreciated.

**Comment by Izumi-kun** [comment]
Response actually can be captured at this point: https://github.com/IMSGlobal/LTI-Tool-Provider-Library-PHP/blob/c9cbfdd5e22b0bc692335b2fd215e8662097ac4c/src/HTTPMessage.php#L136 or
https://github.com/IMSGlobal/LTI-Tool-Provider-Library-PHP/blob/c9cbfdd5e22b0bc692335b2fd215e8662097ac4c/src/HTTPMessage.php#L167

I confirm that current HTTP client implementation is unable to provide useful debug info about failed requests.

**Comment by gtheilman** [comment]
Thank you for the help.

I was able to monitor the curl exchange between my tool provider and Blackboard.   Blackboard was complaining about being "unable to get local issuer certificate".  I was able to find instructions for [downloading and installing the certificate](https://github.com/yabacon/paystack-php/wiki/cURL-error-60:-SSL-certificate-problem:-unable-to-get-local-issuer-certificate-(see-http:--curl.haxx.se-libcurl-c-libcurl-errors.html)) and now both Blackboard and Canvas instances hosted by Instructure are accepting scores.
